app/assets/javascripts/discourse/tests/integration/components/user-menu/menu-test.js

```diff
@@ -86,7 +86,39 @@ module("Integration | Component | user-menu", function (hooks) {
     );
   });
 
+  test("reviewables tab is shown if currentUser can review", async function (assert) {
```

Comment:
```suggestion
  test("reviewables tab is shown if current user can review", async function (assert) {
```

`currentUser` is an internal implementation detail.